Merge s3 fix

Tathanen opened the pull request.

cantyjeffrey commented:
@Tathanen @vokal/web — This should fix the issues with S3 file uploads. Tested locally with `npm link` and seems to be working fine now.

Tathanen commented:
LGTM, let's get this chuggin.

CMDadabo commented:
And thanks for fixing this 👍

Tathanen committed:
Merge pull request #5 from keystonejs/master

Pull in upstream

jimulle committed:
add alt test to delete button

jimulle committed:
documentation updates

stennie committed:
Fix #4815: Typo in Getting started part 4 tutorial

stennie committed:
Fix #4818: Add redirect for /guide/config

autoboxer committed:
Merge branch 'master' into add-alt-text-to-delete-button

autoboxer committed:
Merge pull request #4813 from jimulle/add-alt-text-to-delete-button

add alt text to delete button

Commit:
DOCS: Fix typo for TextArray Field

Tathanen committed:
Merge pull request #7 from keystonejs/master

Update to v4 final

Vultraz committed:
Fixed incorrectly named API method label (fixes #4825)

laurenskling committed:
Update Underscore-Methods.md

underscore instead of undercore

laurenskling committed:
Update options.md

Link is broken without `http://`

autoboxer committed:
Merge pull request #4828 from laurenskling/patch-1

Update Underscore-Methods.md

autoboxer committed:
Merge branch 'master' into patch-2

autoboxer committed:
Merge pull request #4829 from laurenskling/patch-2

Update options.md

autoboxer committed:
Merge branch 'master' into patch-1

autoboxer committed:
Merge pull request #4826 from Vultraz/patch-1

Fixed incorrectly named API method label (fixes #4825)

clement-is committed:
docs(syntax): update an upper-case (#4810)

By copying and pasting the site, I realized that the list was previously referenced with an uppercase.

Commit:
incorrect link to database docs (#4833)

Commit:
fix: fields explorer invalid capitilisations (#4834)

alsoicode committed:
Bug - Fix UTC DateColumn vaue display (#4841)

Added toMoment method to allow Moment.js to use utc dates.
Updated getValue to use toMoment for proper output.

Vultraz committed:
Fixed typo in paginate() docs (#4839)

Commit:
Refer to the newest release "4.0.0"

Probably forgot to update part of readme from the older release "4.0.0.rc.1"

agmt5989 committed:
Completed opening sentence

The sentence "Once you retrieve a list from Keystone, the mongoose methods can be accessed from." seemed incomplete

marcin-lawrowski committed:
Added option to hide Today button in Date field (#4869)

marcin-lawrowski committed:
Fixed dependsOn setting when applied to a boolean field set to false by default (#4867)

Tathanen committed:
Merge branch 'master' of https://github.com/keystonejs/keystone

Tathanen committed:
Build S3 upload name when not present
